deployment/src/main/terraform/main.tf

```
@@ -21,11 +21,11 @@ module s3_buckets {
 }
 
 module storage {
-  source                         = "./modules/storage"
-  data_query_results_store_name  = data.external.secrets.result["data_query_results_store"]
-  raw_ais_store_name             = data.external.secrets.result["raw_ais_store"]
-  ais_data_upload_store_name     = data.external.secrets.result["ais_data_upload_store"]
-  raw_partitioned_ais_store_name = data.external.secrets.result["raw_partitioned_ais_store"]
+  source                               = "./modules/storage"
+  raw_ais_store_name                   = data.external.secrets.result["raw_ais_store"]
+  ais_data_upload_store_name           = data.external.secrets.result["ais_data_upload_store"]
+  raw_partitioned_ais_store_name       = data.external.secrets.result["raw_partitioned_ais_store"]
+  resampled_partitioned_ais_store_name = data.external.secrets.result["resampled_partitioned_ais_store_name"]
```

Inconsistent naming of the secret result?

```suggestion
  resampled_partitioned_ais_store_name = data.external.secrets.result["resampled_partitioned_ais_store"]
```